form.html

```diff
@@ -8,7 +8,7 @@
     <link type="Stylesheet" href="./assets/css/style.css">
     <title>Mystery Melody</title>
 </head>
-
+<center><h1>Questions</h1></center>
```

Comment:
Have you tried centering the headline on the css page? The thing is idk if Mohammed will take off points for doing stylings on the html directly